.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /
+  schedule:
+    interval: weekly
```

Comment:
Weekly is probably rare enough, but we can consider increasing this interval to monthly to lower the noise created by dependabot commits.

Reply:
I doubt there will be much noise as the number of dependencies is quite low